vue-components/src/components/LookupMenu.vue

```diff
@@ -52,6 +57,10 @@ export default Vue.extend( {
 			type: Array,
 			default: (): [] => [],
 		},
+		boldTitles: { // FIME: are we sure we need a property here and its not just a function of a description present?
```

Comment:
I agree and I think this should be implicit in this case, so that the menu appears according to what kind of Menu Items it receives.

Reply:
Making the style of the item labels (they're technically not titles) dynamic, depending on a function of whether a description is present, could then generate menus where some labels are bolded and some aren't, which could communicate a false hierarchy and be confusing for users (and even impact findability?). 

This is I would rather go for what this PR does: giving implementers the freedom to decide whether they want to make the labels more distinguishable in case they know a description will be present.

Reply:
Thank you for clarifying it :)